test/cql-pytest/cassandra_tests/validation/operations/select_multi_column_relation_test.py

```diff
@@ -66,7 +66,7 @@ def testMultiClusteringInvalidQueries(cql, test_keyspace):
         assertInvalidMessage(cql, table, "Invalid null value",
                              "SELECT * FROM %s WHERE a = 0 AND (b, c, d) IN ((?, ?, ?))", 1, 2, None)
         # Reproduces #13217
-        assertInvalidMessage(cql, table, "Invalid null value in condition for columns: [b, c, d]",
+        assertInvalidMessage(cql, table, "Invalid null value in condition for column d",
```

Comment:
You can't change the message like that, it will make this test fail in Cassandra.
If you believe (like I think is the case here) that both messages are fine, please either allow both or simplest - replace the string with a common string that wil pass on both - e.g., "Invalid null value in condition for column".